go/kbfs/simplefs/archive.go

```diff
@@ -278,23 +283,16 @@ func (m *archiveManager) doIndexing(ctx context.Context, jobID string) (err erro
 		return err
 	}
 
-	manifest := make(map[string]keybase1.SimpleFSArchiveFile)
-	gotList := false
-loopReadList:
-	for {
-		listResult, err := m.simpleFS.SimpleFSReadList(ctx, opid)
-		if err != nil || len(listResult.Entries) == 0 {
-			if gotList {
-				break loopReadList
-			}
-		}
-		gotList = true
+	listResult, err := m.simpleFS.SimpleFSReadList(ctx, opid)
+	if err != nil {
+		return err
+	}
```

Comment:
In `keybase fs ls` this calls `SimpleFSReadList` in a loop until error from the call. I found a bug in my copy-paste but then realized the result is written in only once, which makes the loop useless. @strib Do you remember why it was in a loop?

Reply:
Looks like Steve wrote that code originally so I'm not sure, but I'd guess it's built to handle partial lists being returned in case we wanted to support pagination there.  But none of that is implemented as far as I can tell, so a single call after the wait should be good enough if you prefer.